libraries/chain/include/graphene/chain/config.hpp

```diff
@@ -53,6 +53,8 @@
 #define GRAPHENE_MIN_UNDO_HISTORY 10
 #define GRAPHENE_MAX_UNDO_HISTORY 10000
 
+#define GRAPHENE_MAX_NESTED_OBJECTS (200)
```

Comment:
@srpatel19590  - provide details of why the nested objects are added.

Reply:
It is not required. I've already removed this in my next commit.

Reply:
Ok